### DISCUSSION TITLE
QUESTION: is there a way to pin table headers while adding rows?

### DISCUSSION THREAD

**Discussion by wgross** [spectreconsole/spectre.console#1331] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I trying to make a status display for a console application scanning files in a directory. 
The files (and some processing status) are added to a table with a LiveView. The table rows exceed the visible console size.

**Describe the solution you'd like**
I would like to pin the table headers on top of the table. The table should "scroll" under the table headers while adding new rows. Inserting new rows at position 0 would also be  OK if the table header s remain in place.
 
**Describe alternatives you've considered**
I tried to add it to a layout but this didn't fix the missing headers.

I'm also ok with removing table lines at the beginning before adding to the end to keep the table within the console view but Id don't know either how to approach this with Spectre.Console. 

May someone has sovted this problem already and give me a hint?

**Comment by patriksvensson** [comment, marked as answer]
Check out the `LiveTable` example at https://github.com/spectreconsole/spectre.console/tree/main/examples/Console/LiveTable or clone the Spectre.Console repository and run `dotnet tool restore` followed by `dotnet example LiveTable`
